cmd/lakectl/cmd/root_test.go

```diff
@@ -71,58 +71,45 @@ func TestLakectlUnixPerm(t *testing.T) {
 }
 
 func TestInitConfig_LoadingScenarios(t *testing.T) {
-	type setupFn func(t *testing.T) (cleanup func())
-
 	tests := []struct {
 		name        string
-		setup       setupFn
+		setup       func(t *testing.T)
 		expectedURL string
 	}{
 		{
 			name: "using --config flag",
-			setup: func(t *testing.T) func() {
+			setup: func(t *testing.T) {
```

Comment:
relevant to all tests: the func you run as part of the test is an helper, call t.Helper() in the first line.
it will remove the call from the stack report in case of error report - in these tests, when we fail to write to a file.